src/store.js

```diff
@@ -0,0 +1,100 @@
+// Store Class: Handles Storage
+class Store {
+  // Get Tasks : gets the tasks from store
+  static getTasks() {
+    let tasks;
+    if (localStorage.getItem('tasks') === null) {
+      // if there is nothing in storage
+      tasks = []; // create empty array
+    } else {
```

Comment:
[Required] Here you are writing a lot of lines of code to perform one operation. You can use a single line to achieve this 
Ex: 
`const tasks= localStorage.getItem('tasks') || [];`

And then return the tasks.